src/styles.css

```diff
@@ -0,0 +1,1174 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500&family=Poppins:wght@500&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+}
+
+body {
+  background-color: #e5e5e5;
+}
+
+.web-nav {
+  display: none;
+}
+
+.Mylogo {
+  text-decoration: none;
+  color: #6070ff;
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  height: 50px;
+  padding: 2%;
+  background-color: #fff;
+  color: #6070ff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  align-items: center;
+  text-align: center;
+  overflow: hidden;
+}
+
+.header-container #myLinks {
+  display: none;
+}
+
+.nav-item1 {
+  flex-grow: 1;
+  transition: all 1s;
+}
+
+.nav-item1:hover {
+  transform: rotate(360deg);
+}
+
+.nav-item2 {
+  flex-grow: 8;
+}
+
+.nav-item3 {
+  flex-grow: 1;
+}
+
+.overlay {
+  height: 100%;
+  width: 100%;
+  display: none;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  background-color: #6070ff;
+  opacity: 0.9;
+}
```

Comment:
- [ ] kindly consider adding some padding on this overlay to meet the given design check on the below images to get it clearly
- your design 👇 
![yr](https://user-images.githubusercontent.com/60146030/225132556-81b42ce7-b8f6-4990-a0f0-03fa07bf635e.PNG)
- Figma design 👇 
![fg](https://user-images.githubusercontent.com/60146030/225132682-ca0fa13e-38c6-4a70-955d-5c262494c047.PNG)
 and this will also help you in your future job to always deliver what was expected